server/channels/db/migrations/mysql/000119_remove_orphan_draft.up.sql

```diff
@@ -0,0 +1,15 @@
+-- Remove drafts that are not referenced by any other post.
+DELETE FROM drafts
```

Comment:
@Utsav-Ladani Sorry I didn't think this through, this actually should a batch migration. We don't want to lock the table during migration as that could cause bad behaviour for users.

Instead, we can do a batch migration as seen here (we've already done this for drafts for a different reason :P)
https://github.com/mattermost/mattermost/blob/master/server/channels/jobs/delete_empty_drafts_migration/delete_empty_drafts_migration.go

Reply:
Oh, I forgot about that thing. Now, I fixed it in the latest commit. Please let me know if I still need to correct it.